foundation/foundation-mda/src/main/resources/templates/general-docker/mlflow.lineage.properties.vm

```diff
@@ -1,2 +1,3 @@
 data.lineage.enabled=true
-data.lineage.producer=${scmUrl}
\ No newline at end of file
+data.lineage.producer=${scmUrl}
+data.lineage.namespace=default
```

Comment:
Q: Will is break model lineage in existing projects if they fail to add this new property? Might need to add to release notes. 

Also below in [training.base.py.vm](https://github.com/boozallen/aissemble/pull/428/files#diff-ab1998bd6ea1c97c36eedefef420e66a7c458bd28f078d525032b361cbaccb0d) we set the default namespace to the pipeline name. Why do we need to set it here too?

Reply:
This is a miss for the setting default namespace functionality per OpenLineage expected usage. We had changes made for the pyspark pipeline but missed the model training.  
I suspect there are existing project using it because it should be broken right now because of the miss. The new add on property and changes will fix the code.

Yeah.. I will add to the release note

Reply:
Is this namespace different from the namespace set in [training.base.py.vm](https://github.com/boozallen/aissemble/pull/428/files#diff-ab1998bd6ea1c97c36eedefef420e66a7c458bd28f078d525032b361cbaccb0d)?

Reply:
Also `data.lineage.namespace` is the legacy property. If we're adding a new one, it should be in `data.lineage.${pipelineStep}.namespace` format.

Reply:
yeah.. this is a customizable value. We can either set a default value for them or we can just add in the release doc and have them set it by themselves to match the pipeline name or whichever value the prefer.

Reply:
Discussed offline, this namespace config is needed to set the default for the dataset lineage object. Without it the pipeline will fail to run.